test_n13_api.py

```diff
@@ -0,0 +1,25 @@
+import requests
+
+
+def test_httpbin_post(config):
+    mydata = [{'name': 'Nikolay'}, 'hello']
+    r = requests.post(config['base_url'] + 'post', json=mydata)
+    assert r.status_code == 200, r.text
+    data = r.json()
+    assert data['json'] == mydata
```

Comment:
add elapsed time check here too for consistency

Reply:
@nicolas13sochi this is last thing to address and we can merge!

Reply:
well ideally need to add more tests for other http methods and verify some data from headers, but we can do it in a separate PR